src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraGlobalUIConfig.java

```diff
@@ -35,22 +35,16 @@
 @Component
 public class JiraGlobalUIConfig extends UIConfig {
     public static final String LABEL_URL = "Url";
-    public static final String LABEL_USER_NAME = "Username";
-    public static final String LABEL_ACCESS_TOKEN = "Access Token";
+    public static final String LABEL_USER_NAME = "Admin Username";
+    public static final String LABEL_ACCESS_TOKEN = "Admin Api Token";
```

Comment:
Can we also update the variable name?

Reply:
Let's also be consistent with the string "api". Should it be "API", "Api", or "api"?

Reply:
It should be API